src/Schema.php

```diff
@@ -26,6 +26,9 @@ public static function activate() {
 
     // Rename /wp-login.php into /login.php and deny access to XML-RPC API.
     static::ensureFrontControllerAccess();
+
+    // Set assets cache expiry date.
+    static::setAssetsExpiryDate();
```

Comment:
```suggestion
    // Force a long client-side caching time for assets with "ver" query string.
    static::ensureAssetsCacheControl();
```

Reply:
Please also use core-standards:assets-cache-control as the delimiter in the .htaccess file and for the template filename.